.github/ci-scripts/job_runner.py

```diff
@@ -1,12 +1,14 @@
 # /// script
 # requires-python = ">=3.12"
-# dependencies = []
+# dependencies = ["ray[default]"]
```

Comment:
This is so you can run the script locally using `uv`. I.e.:

```sh
uv run .github/ci-scripts/job_runner.py --working_dir=... --entrypoint_script=... --entrypoint_args=...
```

Of course, most people won't directly interface with this script, but rather interface with the `.github/workflows/run-cluster.yaml` workflow, which then spins up a ray-cluster and calls this script.